modules/github_runner_container_app_jobs/base.md

```diff
@@ -22,22 +22,22 @@ resource "azurerm_subnet" "example" {
 }
 
 module "hello-modules_container-apps-gh-runners" {
-  source                   = "Altinn/altinn-modules/azurerm//modules/github_runner_container_app_jobs"
-  version                  = "0.0.1" #See releases for latest version
-  app_id                   = "321321321"
-  install_id               = "123123123"
-  app_key                  = "PHNlY3JldC1hcHAta2V5Pgo="
-  repos                    = [
+  source     = "Altinn/altinn-modules/azurerm//modules/github_runner_container_app_jobs"
+  version    = "1.0.1" #See releases for latest version
```

Comment:
Should this be 1.0.0?

Reply:
Good catch, but I think 1.1.0 would be better as when both the current PRs are merged the version would be 1.1.0, since we added a new submodule. The whole repo with all submodules needs to be versioned as one.